homework/81013/go2html/CommonTypes.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
```

Comment:
Why not include our header?

Reply:
I don't know, actually.
I thought I might want to do something differently. And at first I did (tho it never saw github) but apparently I left it as yours at the end.